Packs/CommonScripts/Scripts/SearchIncidentsV2/README.md

```diff
@@ -60,31 +60,32 @@ Spring Core and Cloud Function SpEL RCEs
-| level              | Filter by Severity                                                                                                                                                                                                                                   |
-| owner              | Filter by incident owners                                                                                                                                                                                                                            |
-| details            | Filter by incident details                                                                                                                                                                                                                           |
-| type               | Filter by incident type                                                                                                                                                                                                                              |
-| query              | Use free form query \(use Lucene syntax\) as filter. All other filters will be ignored when this filter is used.                                                                                                                                     |
-| page               | Filter by the page number (deprecated)                                                                                                                                                                                                               |
-| trimevents         | The number of events to return from the alert JSON. The default is 0, which returns all events.<br/>Note that the count is from the head of the list, regardless of event time or other properties.                                                  |
-| size               | Number of incidents per page \(per fetch\) (deprecated)                                                                                                                                                                                              |
-| sort               | Sort in format of field.asc,field.desc,...                                                                                                                                                                                                           |
-| searchresultslabel | If provided, the value of this argument will be set under the searchResultsLabel context key for each incident found.                                                                                                                                |
+| **Argument Name**  | **Description**                                                                                                                                                                                                                                    |
```

Comment:
Why was all the table changed please revert all changes that are not related to your modification.